package.json

```diff
@@ -1,8 +1,12 @@
 {
   "name": "react-native-kontaktio",
-  "version": "2.10.0",
+  "version": "3.0.0",
   "description": "React-native module for detecting [Kontakt.io](http://kontakt.io/) beacons. You have to own some Kontakt.io beacons, configure them via their managment console and have your api-key handy.",
-  "main": "index.js",
+  "main": "lib/index.js",
+  "types": "lib/types.d.ts",
+  "files": [
```

Comment:
Given this line, we don't need the `.npmignore` file anymore because the `files` property defines, which files get uploaded to the npm registry.

Reply:
Ok, I haven't updated it yet but just added some files here below